transit_odp/organisation/csv/organisation.py

```diff
@@ -624,6 +670,11 @@ def _get_organisation_details_dataframe() -> DataFrame:
         .values(*ORG_USER_FIELDS)
     )
 
+    if is_fares_validator_active:
+        orgs["archived_matching_report_url"] = orgs.apply(
+            lambda x: get_archived_matching_report_url(x), axis=1
+        )
+
```

Comment:
I think you should put this part of the code inside `_prepare_calculated_columns()` function.

Reply:
@surabhika I've pushed this change.

Reply:
Could you please increment the version to get this into dev?

Reply:
@surabhika Incremented the version number

Reply:
I don't think you've pushed it?

Reply:
yhh it wasn't showing up for some reason, it should have come through now